snippets/card-collection.liquid

```diff
@@ -23,16 +23,16 @@
   endif
 -%}
 
-<div class="card-wrapper animate-arrow">
+<div class="collection-card-wrapper card-wrapper animate-arrow">
```

Comment:
Same suggestion as above but with `card-wrapper--collection` instead of `collection-card-wrapper`

```suggestion
<div class="card-wrapper card-wrapper--collection animate-arrow">
```

We'd of course need to update the `base.css` selector from `.collection-card-wrapper .card` to `.card-wrapper--collection .card`